run-tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
You can mention this script int the README.md

Reply:
added